docs/en/operations/system-tables/zookeeper_log.md

```diff
@@ -0,0 +1,132 @@
+# system.zookeeper_log {#system-zookeeper_log}
+
+This table contains information about the parameters of the request to the ZooKeeper client and the response from it.
+
+For requests, only columns with request parameters are filled in, and the remaining columns are filled with default values (`0` or `NULL`). When the response arrives, the data from the response is added to the other columns.
+
+!!! info "Note"
+    The table does not exist if ZooKeeper is not configured.
```

Comment:
It does. But if ClickHouse is configured to use TestKeeper (instead of ZooKeeper client) then nothing is logged.